app/helpers/calendar_helper.rb

```diff
@@ -0,0 +1,24 @@
+module CalendarHelper
+
+  def get_event(day, time)
+    datetime = add_date_and_time(day, time)
+    @events[day.wday].find do |event|
+      event.start_time == datetime
+    end if @events[day.wday]
+  end
+
+  def event_ongoing?(day, time)
+    datetime = add_date_and_time(day, time)
```

Comment:
It might make sense to do "return early" here, otherwise the value returned by the method will be datetime 
`return false unless @events[day.wday]`